packages/styles/src/CssBaseline.ts

```diff
@@ -76,6 +76,16 @@ const baseline = {
     color: "inherit",
     textDecoration: "inherit",
   },
+
+  ".uikit-root-element": {
+    backgroundColor: "var(--uikit-colors-backgroundColor)",
+    accentColor: "var(--uikit-colors-secondary)",
+    color: "var(--uikit-colors-secondary)",
+    fontSize: "var(--uikit-typography-body-fontSize)",
+    fontWeight: "var(--uikit-typography-body-fontWeight)",
+    lineHeight: "var(--uikit-typography-body-lineHeight)",
+    letterSpacing: "var(--uikit-typography-body-letterSpacing)",
```

Comment:
These CSS var names are internal. We should use the `theme` object instead (shorter, type-safe, future-proof).
```suggestion
    backgroundColor: theme.colors.backgroundColor,
    accentColor: theme.colors.secondary,
    color: theme.colors.secondary,
    fontSize: theme.typography.body.fontSize,
    fontWeight: theme.typography.body.fontWeight,
    lineHeight: theme.typography.body.lineHeight,
    letterSpacing: theme.typography.body.letterSpacing,
```